markdown/usage/troubleshooting.md

```diff
@@ -3,33 +3,88 @@ title: Troubleshooting
 3. When using the pipeline with paired end data, the path must use `{1,2}` or `{R1,R2}` notation to specify read pairs.
-4. If you are running Single end data make sure to specify `--singleEnd`
+    - This notation is interpreted by Nextflow to mean anything with the same string other than R1 and R2 in the file name, will be be assumed to be a pair of files.
+4. If you are running single-end data make sure to specify `--singleEnd`
+5. [Your data should be organised in a 'tidy' manner](#data-organization)
 
-If the pipeline can't find your files then you will get the following error
+Note that if your sample name is "messy" then you have to be very particular with your glob specification. A file name like `L1-1-D-2h_S1_L002_R1_001.fastq.gz` can be difficult enough for a human to read. Specifying `*{1,2}*.gz` wont work give you what you want, whilst `*{R1,R2}*.gz` will.
+
+If you are using a sample sheet or TSV input method, check there is not a mistake or typo in the path in a given column. Common mistakes ar e a trailing space at the end of the path, which can cause problems.
+
+### Output for only a single sample although I specified multiple with wildcards
+
+You must specify paths to files in quotes, otherwise your _shell_ (e.g. bash) will evaluate any wildcards (\*) rather than Nextflow.
+
+For example:
 
 ```bash
-ERROR ~ Cannot find any reads matching: *{1,2}.fastq.gz
+nextflow run nf-core/eager --input /path/to/sample_*/*.fq.gz
+```
+
+Maybe evaluated by your shell as:
+
+```bash
+nextflow run nf-core/eager --input /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz
+```
+
+And Nextflow will only take the first path after `--input`, ignoring the others.
+
+On the other hand, encapsulating the path in quotes will allow _Nextflow_ to evaluate the paths.
+
+```bash
+nextflow run nf-core/eager --input "/path/to/sample_*/*.fq.gz"
 ```
 
-Note that if your sample name is "messy" then you have to be very particular with your glob specification. A file name like `L1-1-D-2h_S1_L002_R1_001.fastq.gz` can be difficult enough for a human to read. Specifying `*{1,2}*.gz` wont work give you what you want Whilst `*{R1,R2}*.gz` will.
+### Data organization
 
-## Data organization
 The pipeline can't take a list of multiple input files - it takes a glob expression. If your input files are scattered in different paths then we recommend that you generate a directory with symlinked files. If running in paired end mode please make sure that your files are sensibly named so that they can be properly paired. See the previous point.
 
+## The pipeline crashes almost immediately with an early pipeline step
+
+Sometimes a newly downloaded and set up nf-core pipeline will encounter an issue where a run almost immediately crashes (e.g. at `fastqc`, `output_documentation` etc.) saying the tool could not be found or similar.
```

Comment:
Maybe add a section that says something about HPC job schedulers? Or more generally - suggesting that people look at the error log and don't get distracted by the actual process that failed 🙄

Reply:
Not sure what you mean by HPC job schedulers?

Reply:
slurm and similar schedulers I'd say

Reply:
Pipeline could crash because of not enough memory, cpus, or other restrictions due to the scheduler itself

Reply:
Ok, I've not had that issue recently so can't give more specific advice/examples what that looks like, but added something for now we can expand late.r

Reply:
I'll try to fake it

Reply:
```
Error executing process > 'Get_software_versions'

Caused by:
  Failed to submit process to grid scheduler for execution

Command executed:

  sbatch .command.run

Command exit status:
  1

Command output:
  sbatch: error: Batch job submission failed: Invalid account or account/partition combination specified

Work dir:
  /domus/h1/maxime/work/1e/e9f5eba379b65d7a37138e3b27fb93
```